packages/pulpcore/python-tablib/python-tablib.spec

```diff
@@ -0,0 +1,79 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name tablib
+
+Name:           python-%{pypi_name}
+Version:        1.1.0
+Release:        1%{?dist}
+Summary:        Format agnostic tabular data library (XLS, JSON, YAML, CSV)
+
+License:        MIT
+URL:            https://tablib.readthedocs.io
+Source0:        https://files.pythonhosted.org/packages/source/t/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-markuppy
+BuildRequires:  python3-markuppy
```

Comment:
Odd that so many lines are duplicated

Reply:
Yeah. It's because tablib actually mentions those multiple times in `setup.py` :(